apps/src/templates/instructions/MarkdownInstructions.jsx

```diff
@@ -99,18 +100,19 @@ const MarkdownInstructions = React.createClass({
       noInstructionsWhenCollapsed
     } = this.props;
 
-    // In cases where we have an image, we want to guarantee a certain amount of
-    // height, to deal with the fact that we want know how much height the image
-    // actually needs until it has loaded
-    const hasImage = /<img src/.test(renderedMarkdown);
+    // In cases where we have a full-size image (as opposed to the inline images we use in
+    // Star Wars), we want to guarantee a certain amount of height, to deal with the fact
+    // that we won't know how much height the image actually needs until it has loaded
+    const hasFullSizeImage = !this.props.hasInlineImages && /<img src/.test(renderedMarkdown);
```

Comment:
Not currently; we only use inline images in star wars, for which our "instructions" are only the relatively short quotes from Rey or Leia.